VirtoCommerce.Platform.Web/wwwroot/js/app/exportImport/exportNew.js

```diff
@@ -0,0 +1,28 @@
+angular.module('platformWebApp')
```

Comment:
We should not add module menu items in platform code.
There is a service that allows to add buttons which could be used in module - [example for Pricing](https://github.com/VirtoCommerce/vc-platform-core/blob/dev/Modules/vc-module-pricing/VirtoCommerce.PricingModule.Web/Scripts/pricing.js#L34)